src/lib/theme/theme.ts

```diff
@@ -9,6 +9,7 @@ export interface Colors {
 
   // text
   primary: string
+  accentText: string
```

Comment:
Call this `accent` - none of the other properties are suffixed with `text`.

Reply:
hmm...`accent` is taken. Maybe `contrast`?

Reply:
this is specifically text to be overlayed over `accent`.

Reply:
tbh I don't think we should make this change (@ppphhhiiilll )

black is a higher contrast text over the pink we are using, and I think we should make all text just be `readableColor(<backgroundColor>)`

Reply:
> this is specifically text to overlay `accent`.

In that case, can you call it `onAccent`?